image.go

```diff
@@ -421,6 +421,7 @@ type BuildImageOptions struct {
 	AuthConfigs         AuthConfigurations `qs:"-"` // for newer docker X-Registry-Config header
 	ContextDir          string             `qs:"-"`
 	Ulimits             []ULimit           `qs:"-"`
+	BuildArgs           []BuildArg         `qs:"buildarg"`
```

Comment:
Since you're manually adding this to the querystring, please set the qs tag to "-":

```
BuildArgs []Buildarg `qs:"-"`
```